config/routes.rb

```diff
@@ -1,7 +1,8 @@
+require 'repost'
 Rails.application.routes.draw do
   root to: 'home#show'
 
-  get 'login', to: redirect('/auth/google_oauth2'), as: 'login'
+  get 'login', to: redirect_post('/auth/google_oauth2'), as: 'login'
```

Comment:
This will need to point to the new login method, not do this redirect.
So like `to: '?#login'`.  But I'm not sure what it should look like for a controller action that is on the application controller.  You might need to move it to one of the other ones like home or session